ts/src/btcturk.ts

```diff
@@ -758,8 +758,8 @@ export default class btcturk extends Exchange {
             request['pairSymbol'] = market['id'];
         }
         const response = await this.privateGetOpenOrders (this.extend (request, params));
-        const data = this.safeValue (response, 'data');
-        const bids = this.safeValue (data, 'bids', []);
+        const data = this.safeList (response, 'data', []);
```

Comment:
@AresArtemius u sure this is a list? because on the line below, we do `this.safeList (data, 'bids', []);` using a string key so seems off

Reply:
@carlosmiei Yep, sorry, made a mistake. safeDIct there